test/unit/visual/cases/type.js

```diff
@@ -1,6 +1,6 @@
 import { visualSuite, visualTest } from "../visualTest";
 
-visualSuite("Typography", function () {
```

Comment:
Maybe let's keep the title "Typograhy" for now so we won't have to move all the existing screenshots to a new folder